Tasks/MavenAuthenticateV0/mavenauth.ts

```diff
@@ -54,6 +50,49 @@ async function run(): Promise<void> {
             tl.debug(tl.loc("Info_CreatingSettingsXml", userSettingsXmlPath));
             tl.setVariable('FIRST_RUN_SETTINGS_XML_EXISTS_PATH', userSettingsXmlPath);
         }
+        
+#if WIF
+        const feedUrl = tl.getInput("feedUrl");
+        const entraWifServiceConnectionName = tl.getInput("workloadIdentityServiceConnection");
+        const feedName = tl.getInput('feedName');
+
+        if (feedUrl && entraWifServiceConnectionName && feedName) {
```

Comment:
Can we programatically get the feedname from the url instead? I am worried that we will get issues from users misconfiguring the name/url

Reply:
Looks like we dont have any common code to retrieve feedname using an URL. It is possible that the user will pass in an URL that's using GUID or some other format and it should still work since the token generation doesn't rely on the URL format. Maybe it's not a bad idea to ask for FeedName. We can't do a drop down list since the feed may not be in the same org.

Reply:
I think we have an example in npmrc: https://github.com/microsoft/azure-pipelines-tasks-common-packages/blob/1060920e4c438d15e393e1d9f5af647a6c435d00/common-npm-packages/packaging-common/npm/npmutil.ts#L40 but yes there is no common code.

Reply:
Yeah I guess that could work. Feed name will be in between _packaging and {protocolName}

Reply:
Hey, I added parsing for the feed name